### DISCUSSION TITLE
How to run neural network on RISC-V devices?

### DISCUSSION THREAD

**Discussion by FlintWangacc** [iree-org/iree#17697] [General]
I use ```iree-import-tf``` to import a pb file and produce ```iree_input.mlir```
```shell
iree-import-tf --tf-import-type=savedmodel_v1 --tf-savedmodel-exported-names=predict ./ -o iree_input.mlir
```

and then I use ```iree-compile``` to build vmfb file
```shell
../iree-build/install/bin/iree-compile --iree-hal-target-backends=vmvx ../../AI/tensorflow/iree_input.mlir -o ${HOME}/tmp/test.vmfb
```
But how to run this test.vmfb file on RISC-V device?

**Comment by ScottTodd** [comment]
These may help:
* https://iree.dev/building-from-source/riscv/
* https://iree.dev/guides/deployment-configurations/bare-metal/
* https://github.com/iree-org/iree/blob/main/build_tools/cmake/riscv.toolchain.cmake

**Reply by FlintWangacc** [reply]
Thank you

**Reply by FlintWangacc** [reply]
The problem is solved.
